app/controllers/api/v1/members_controller.rb

```diff
@@ -26,14 +26,32 @@ def update
 
     api :DELETE, '/organizations/:organization_id/members/:id', 'Destroy member'
     def destroy
-      authorize @member
       @member.destroy
     end
 
+    api :GET, '/organizations/:organization_id/members/:id', 'Return member'
+    def show
+    end
+
+    api :GET, '/organizations/:organization_id/member_info', 'Return current member'
+    def current
+      @member = current_member
+    end
+
+    api :PUT, '/organizations/:organization_id/members/:id/update_last_viewed_at', 'Update last viewed at'
```

Comment:
@yunixon there is no reasons to include `:id` in url, because user may update only himself `last_viewed_at`. I think you should add separate url to work with current member only which will not include `:id` (may be `current_member`?). And `update_last_viewed_at` is bad name for path (but not for action). Path should not include verb word. May be PUT `last_visit` will work. Not sure.